docs/_docs/code-snippets/java/src/main/java/org/apache/ignite/snippets/TDE.java

```diff
@@ -58,6 +58,13 @@ void configuration() {
         IgniteFuture<Void> future = ignite.encryption().changeMasterKey("newMasterKeyName");
         //end::master-key-rotation[]
 
+        //tag::cache-group-key-rotation[]
+        // Starts cache group encryption key change process.
+        // This future will be completed when the new encryption key is set for writing on
```

Comment:
```suggestion
        // This feature will be completed when the new encryption key is set for writing on
```

Reply:
Thanks, but this is not a typo, it refers specifically to "future" and not a "feature"